integration-healthcheck/src/main/java/org/symphonyoss/integration/healthcheck/services/KmAuthHealthIndicator.java

```diff
@@ -14,10 +14,16 @@
 public class KmAuthHealthIndicator extends AuthenticationServiceHealthIndicator {
 
   private static final String SERVICE_FIELD = "keyauth";
+  public static final String FRIENDLY_SERVICE_NAME = "Key Manager Auth";
 
   @Override
   protected ServiceName getServiceName() {
-    return ServiceName.KEY_MANAGER_AUTH;
+    return ServiceName.POD;
+  }
+
+  @Override
+  protected String getFriendlyServiceName() {
+    return FRIENDLY_SERVICE_NAME;
```

Comment:
This method should return ServiceName.KEY_MANAGER_AUTH.toString()

Reply:
Fixed!